agent/src/beerocks/slave/ap_manager_thread.cpp

```diff
@@ -52,6 +52,20 @@ static void copy_radio_supported_channels(std::shared_ptr<bwl::ap_wlan_hal> &ap_
     }
 }
 
+static int8_t get_tx_power(std::shared_ptr<bwl::ap_wlan_hal> &ap_wlan_hal)
+{
+    auto radio_channels = ap_wlan_hal->get_radio_info().supported_channels;
+    uint8_t channel     = ap_wlan_hal->get_radio_info().channel;
+
+    for (uint i = 0; i < beerocks::message::SUPPORTED_CHANNELS_LENGTH && i < radio_channels.size();
```

Comment:
We don't usually use `uint`. 
Any specific reason you decided to not use `uint8_t` here?

Reply:
Changed to uint8_t. It was copy past from upper function.